MinimalApi.Tests.Integration/Tests/ResultEndpoints/WithRequest/ConflictEndpointTests.cs

```diff
@@ -0,0 +1,38 @@
+using MinimalApi.Features.ResultEndpoints.WithRequest;
+
+namespace MinimalApi.Tests.Integration.Tests.ResultEndpoints.WithRequest;
+
+[Collection("Endpoint")]
+public class ConflictEndpointTests(RadEndpointFixture f)
+{
+    [Fact]
+    public async Task When_ItemExists_ReturnsConflictString()
```

Comment:
Won't this be better as a POST request so its simulating the a typical operation like saving a record that already exists?  Update the endpoint and test to reflect a more typical use case for conflict.